main/commons/src/main/java/org/cryptomator/common/mountpoint/TemporaryMountPointChooser.java

```diff
@@ -69,13 +69,20 @@ public boolean prepare(Volume caller, Path mountPoint) throws InvalidMountPointE
 		try {
 			switch (caller.getMountPointRequirement()) {
 				case PARENT_NO_MOUNT_POINT -> {
-					Files.createDirectories(mountPoint.getParent());
-					LOG.debug("Successfully created folder for mount point: {}", mountPoint);
+					//Create everything up to the parent (but not the actual mountpoint)
+					Path parent = mountPoint.getParent();
+					Files.createDirectories(parent);
+
+					//Get the name of the parent directory (#getFileName())
+					//or just use the parent if the name is null (that's the case if parent is the root)
+					Path parentName = parent.getFileName() != null ? parent.getFileName() : parent;
+					LOG.debug("Successfully created/checked parent folder (\"{}\") for mount point: {}", parentName, mountPoint);
```

Comment:
I think we need to have a discussion about clean code practices.

`Files.createDirectories(mountPoint.getParent());` is a perfectly clear, unambiguous and readable line. Why bloating it and introducing comments that just repeat what the method names already say?

While this is already highly debatable and commonly considered a bad practice, as it waters down the effect of comments in places where they _really_ add additional insights, it gets worse:

`parentName` is used _solely_ during debug logging and the log message itself doesn't hold _any_ additional information over the old one.

WHY BLOAT THE CODE BY x4?

Reply:
Because I considered it ambiguous. I will revert.